.github/workflows/release.yaml

```diff
@@ -0,0 +1,43 @@
+name: release
+
+on:
+  push:
+    branches:
+      - '**'
+    tags:
+      - 'v*'
+  pull_request:
```

Comment:
Do we want this to run on pull requests - wouldn't we only release on a tag event? Also, I notice that we are specifying on `push` events but the only thing those trigger is a `checkout`, `install` and `login`.

Reply:
We're running goreleaser on every PR and merge to any upstream branch right now, but updating the goreleaser arguments and logging into quay when a tag event has been detected. Does that make sense? We could also release rukpak when changes land in the main branch too.

Reply:
So we're running goreleaser on all of these events, but it only actually releases on a tag event?

Reply:
Yup that's the correct read.